mods/ra/rules/husks.yaml

```diff
@@ -95,11 +95,11 @@ TRAN.Husk:
 		Offset: 597,0,213
 		Sequence: rotor2
 	RevealsShroud:
-		Range: 10c0
+		Range: 8c0
```

Comment:
Added 8c0 vision range (and the associated `RevealsShroud@GAPGEN`) to Transport.Husk as with the Transport (Chinook) itself. Overlooked by the previous balance PR